itou/job_applications/notifications.py

```diff
@@ -6,6 +6,20 @@
     PrescriberOrEmployerNotification,
 )
 from itou.job_applications.enums import RefusalReason
+from itou.utils.perms.utils import _can_view_personal_information
+
+
+class ProxyNotification(PrescriberOrEmployerNotification, EmailNotification):
+    def get_context(self):
+        context = super().get_context()
+        job_application = context["job_application"]
+        return context | {
+            "can_view_personal_information": _can_view_personal_information(
+                viewer=self.user,
+                user=job_application.job_seeker,
+                viewer_is_prescriber_from_authorized_org=self.user.is_prescriber_with_authorized_org_memberships,
```

Comment:
Je me demande si on ne devrait pas vérifier `job_application.sender_prescriber_organization and job_application.sender_prescriber_organization.is_authorized` ? (et pareil pour diagoriente)

Reply:
Si une personne quitte une organisation, elle perd aussi le bénéfice des habilitations de l’organisation. J’aurais tendance à continuer à me baser sur l’état du destinataire.